jobs/pulse3d/pulse3d-worker/src/main.py

```diff
@@ -72,25 +74,31 @@ async def process(con, item):
             upload_details = await con.fetchrow(query, upload_id)
 
             prefix = upload_details["prefix"]
-            filename = upload_details["filename"]
-            key = f"{prefix}/{filename}"
+            metadata = json.loads(item["meta"])
+            upload_filename = upload_details["filename"]
+            name_override = metadata.get("name_override")
+            # if a new name has been given in the upload form, then replace here, else use original name
+            analysis_filename = (
```

Comment:
Could probably just do something like
`analysis_filename = metadata.get("name_override", upload_filename) + ".zip"`

Reply:
upload_filename already includes the file extension

Reply:
Oh true, could do `analysis_filename = f"{name_override}.zip" if (name_override := metadata.get("name_override")) else upload_filename`

Reply:
@tannermpeterson can you approve again, I updated this var